.github/workflows/lock-issue-pr-with-message.yml

```diff
@@ -0,0 +1,47 @@
+name: Lock Closed Issues and PRs with message
+
+on:
+  pull_request_target:
+    types: [closed]
+  issues:
+    types: [closed]
+
+jobs:
+  auto_comment:
+    permissions:
+      pull-requests: write
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: aws-actions/closed-issue-message@v1
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          message: |
+            Comments on closed issues and PRs are hard for our team to see. 
+            If you need help, please open a new issue that references this one.
+            If you wish to keep having a conversation with other community members under this issue feel free to do so.
```

Comment:
I think this line should be removed, I think community will not be able to add comments

Reply:
sorry I missed this line while reviewing the PR

Reply:
Oh good point. Let me remove it.